cmake-tests/test_find_package/project/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+cmake_minimum_required(VERSION 3.19)
+
+project(UseFindPackageTest)
+
+find_package(launchdarkly REQUIRED)
+
+add_executable(use_find_package_server main_server.cpp)
```

Comment:
You may wonder "why launchdarkly::launchdarkly-cpp-server"? Because when I originally named the targets, I was thinking "global namespace, it might clash with someone's `server` or `client` target."

Not sure how to make this match our `launchdarkly::{client,server}` aliases that are usable in the `add_subdirectory` case.